Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py

```diff
@@ -646,9 +665,11 @@ def get_indicators_command(client: Client, args: Dict[str, str], feed_tags: list
     Returns:
         Demisto Outputs.
     """
-    limit = int(args.get('limit', '10'))
+    limit = arg_to_number(args.get('limit', '10'))
+    if not feed_tags:
+        feed_tags = []
 
-    indicators = client.fetch_stix_objects_from_api(test=True, type='indicator', limit=limit)
+    indicators = client.fetch_stix_objects_from_api(test=True, type='indicator', limit=limit if limit else -1)
```

Comment:
Why do you do limit = -1?

Reply:
this is only due to mypy errors, since i have this line ```limit = arg_to_number(args.get('limit', '10'))``` which returns either Int/None (Optional[int]) - mypy complains on that line because limit could possibly be None.
The reason I am forwarding -1 the ```client.fetch_stix_objects_from_api(test=True, type='indicator', limit=limit if limit else -1)``` is only because the ```limit``` default value for that function is -1